core/base/result.go

```diff
@@ -28,25 +29,38 @@ const (
 	BlockTypeCircuitBreaking
 	BlockTypeSystemFlow
 	BlockTypeHotSpotParamFlow
+	// BlockTypeRegistryStart convenient to expand the type to avoid conflicts
+	BlockTypeRegistryStart
 )
 
+var (
+	blockTypeNames = []string{
```

Comment:
Use Map to store types and descriptions and cancel BlockTypenames. Is that OK?

Reply:
![image](https://user-images.githubusercontent.com/13758306/108507472-55fb0400-72f5-11eb-9ba2-b90ceabb9baf.png)

The performance of map is poor. 
If you feel that the code is inconvenient to maintain, 
modify the version of the "switch",is it ok ?

Reply:
In order to improve the scalability and facilitate the registration of new types, it is modified to map. Although the performance of map is poor, it is acceptable.